cockroachdb/values.yaml

```diff
@@ -236,6 +240,14 @@ service:
     # Additional annotations to apply to this Service.
     annotations: {}
 
+  metrics:
```

Comment:
I'm not convinced we need a separate service for metrics, as opposed to reusing an existing service. But if we do, I think the `metrics.enabled` field (defined below) should be located here instead (or these fields should be moved into the `metrics` section below).